vendor/manifest

```diff
@@ -888,6 +888,14 @@
 			"branch": "v1"
 		},
 		{
+			"importpath": "gopkg.in/urfave/cli.v1",
+			"repository": "https://gopkg.in/urfave/cli.v1",
```

Comment:
I searched the urfave/cli code, itself has this dependency, so I assume I cannot remove this directory simply.